Assets/MixedRealityToolkit.Examples/Demos/EyeTracking/Demo_Visualizer/Scripts/InputPointerVisualizer.cs

```diff
@@ -0,0 +1,457 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Examples.Demos.EyeTracking.Logging
+{
+    /// <summary>
+    /// This visualizer can be used to represent pointer input data, e.g., from a handheld controller,
+    /// from hand, head or eye tracking. In general, it assumes a pointing origin and direction,
+    /// 
+    /// ------------
+    /// *** TODO ***
+    /// ------------
+    /// - Load log files
+    /// - Replace GameObjects with particle systems for higher perf on HoloLens
+    /// - Question: How to visualize data from multiple ppl? Color lookup table? Limit number of scanpaths visible at a time?
+    /// - Question: How can users load files from HoloLens? Built-in file explore? Load by session?
+    /// - Allow for tweaking parameters while viewing the scene, such as "transparency" etc.
+    /// ------------
+    /// </summary>
+    public class InputPointerVisualizer : MonoBehaviour
+    {
+        public enum VisModes
```

Comment:
nit: this name isn't super intuitive. This type also needs summaries.

Reply:
Tracked in https://github.com/Microsoft/MixedRealityToolkit-Unity/issues/3767